locale/fr.po

```diff
@@ -62,55 +63,52 @@ msgid "Start recording"
 msgstr "Démarrer l'enregistrement"
 
 #: extension.js:281 extension.js:613
-#, fuzzy
 msgid " sec. delay before recording"
-msgstr "Retarder le démarrage"
+msgstr " sec. de retardement avant enregistrement"
```

Comment:
I see that this string starts with a space. Is that wanted?

Reply:
Yes because you have the time concatenated in front of it (Ex.:  34 sec. delay before recording). For the space, I still based my self on your MSGID @ line 66.